src/nvim/api/vim.c

```diff
@@ -254,6 +255,25 @@ Object nvim_call_function(String fname, Array args, Error *err)
   return rv;
 }
 
+/// Execute lua code. Parameters might be passed, they are available inside
+/// the chunk as `...`. They chunk can return a value.
+///
+/// To evaluate an expression, it must be prefixed with "return ".
+/// For instance, to call a lua function with arguments and return value,
```

Comment:
"arguments and return value" should have been a noun phrase, but as was ambiguous (even if both readings _kinda_ make sense, as you see) I will rewrite it